virt/virt-4-11-release-notes.adoc

```diff
@@ -80,11 +80,12 @@ The SVVP Certification applies to:
 //CNV-17269 Enable and disabe VM descheduler from UI
 
 //CNV-17805 Single VM Overview page
+* You can access virtual machines by navigating to *Virtualization* -> *VirtualMachines* in the side menu. For each virtual machine, the xref:../virt/logging_events_monitoring/virt-viewing-information-about-vm-workloads.adoc#virt-about-the-vm-dashboard_virt-viewing-information-about-vm-workloads[updated *Overview* tab] now provides information about the virtual machine configuration, alerts, snapshots, network interfaces, disks, usage data, and hardware devices on a single page.
```

Comment:
Is the first sentence referring to a new feature? If not, I might tighten this up by doing something like:

```suggestion
* Each virtual machine now has an xref:../virt/logging_events_monitoring/virt-viewing-information-about-vm-workloads.adoc#virt-about-the-vm-dashboard_virt-viewing-information-about-vm-workloads[updated *Overview* tab] that provides information about the virtual machine configuration, alerts, snapshots, network interfaces, disks, usage data, and hardware devices.
```

(I also removed the "on a single page" part, since that seems implied by the "tab," but that might just be my preference.)